backends/v3/Cargo.toml

```diff
@@ -16,7 +16,7 @@ path = "src/main.rs"
 [dependencies]
 async-trait = "0.1.74"
 async-stream = "0.3.5"
-axum = { version = "0.7", features = ["json"] }
+axum = { version = "0.8", features = ["json"] }
```

Comment:
Why do we need upgrades ?

Reply:
only needed to generate valid openapi docs for the previous approach of `Option<MessageContent>` this is no longer needed with improved typing (using an enum instead of two optionals) in the latest commits